.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: Build
+on: [push, pull_request, workflow_dispatch]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install ocaml ocamlbuild libnum-ocaml-dev autoconf automake indent libtool fig2dev
+
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Bootstrap
+      run: sh bootstrap.sh
```

Comment:
Maybe also `make -C test smallcheck` to run a few tests.

Reply:
After the build step, I mean.

Reply:
(Not sure how long the bootstrap will take on github's CI, but hopefully < 30 min?)

Reply:
The above command fails with:
```
make: *** test: No such file or directory.  Stop.
```

I will look into it later.

Reply:
It was missing an "s".

Done